promgen/serializers.py

```diff
@@ -120,3 +121,53 @@ class HostSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Host
         exclude = ("id", "farm")
+
+
+class MatcherSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    value = serializers.CharField()
+    isRegex = serializers.BooleanField()
+    isEqual = serializers.BooleanField(required=False, default=True)
```

Comment:
`required` is automatically `False` if you define a `default` value, so this is enough:

```
    isEqual = serializers.BooleanField(default=True)
```

Reply:
Fixed in 9d3afcb2a548d5de45f7c6037fe7b97b03a7a52d.